lbrynet/conf.py

```diff
@@ -58,29 +65,72 @@ def _win_path_to_bytes(path):
     return path
 
-if sys.platform.startswith('darwin'):
-    platform = DARWIN
-    default_download_directory = os.path.join(os.path.expanduser('~'), 'Downloads')
-    default_data_dir = user_data_dir('LBRY')
-    default_lbryum_dir = os.path.join(os.path.expanduser('~'), '.lbryum')
-elif sys.platform.startswith('win'):
-    platform = WINDOWS
-    from lbrynet.winhelpers.knownpaths import get_path, FOLDERID, UserHandle
 
-    default_download_directory = get_path(FOLDERID.Downloads, UserHandle.current)
-    default_data_dir = os.path.join(
-        get_path(FOLDERID.RoamingAppData, UserHandle.current), 'lbrynet')
-    default_lbryum_dir = os.path.join(
-        get_path(FOLDERID.RoamingAppData, UserHandle.current), 'lbryum')
+def _get_old_directories(platform):
+    dirs = {}
+    if platform == WINDOWS:
+        dirs['data'] = user_data_dir('lbrynet', roaming=True)
+        dirs['lbryum'] = user_data_dir('lbryum', roaming=True)
+        dirs['download'] = get_path(FOLDERID.Downloads, UserHandle.current)
+    elif platform == DARWIN:
+        dirs['data'] = user_data_dir('LBRY')
+        dirs['lbryum'] = os.path.expanduser('~/.lbryum')
+        dirs['download'] = os.path.expanduser('~/Downloads')
+    elif platform == LINUX:
+        dirs['data'] = os.path.expanduser('~/.lbrynet')
+        dirs['lbryum'] = os.path.expanduser('~/.lbryum')
+        dirs['download'] = os.path.expanduser('~/Downloads')
+    return dirs
+
+
+def _get_new_directories(platform):
+    dirs = {}
+    if platform == WINDOWS:
+        dirs['data'] = user_data_dir('lbrynet', 'lbry')
+        dirs['lbryum'] = user_data_dir('lbryum', 'lbry')
+        dirs['download'] = get_path(FOLDERID.Downloads, UserHandle.current)
+    elif platform == DARWIN:
+        _get_old_directories(platform)
+    elif platform == LINUX:
+        dirs['data'] = user_data_dir('lbry/lbrynet')
+        dirs['lbryum'] = user_data_dir('lbry/lbryum')
+        try:
+            with open(os.path.join(user_config_dir(), 'user-dirs.dirs'), 'r') as xdg:
+                down_dir = re.search(r'XDG_DOWNLOAD_DIR=(.+)', xdg.read()).group(1)
+                down_dir = re.sub('\$HOME', os.getenv('HOME'), down_dir)
+                dirs['download'] = re.sub('\"', '', down_dir)
+        except EnvironmentError:
+            dirs['download'] = os.getenv('XDG_DOWNLOAD_DIR')
+
+        if not dirs['download']:
+            dirs['download'] = os.path.expanduser('~/Downloads')
+    return dirs
+
 
-    default_download_directory = _win_path_to_bytes(default_download_directory)
-    default_data_dir = _win_path_to_bytes(default_data_dir)
-    default_lbryum_dir = _win_path_to_bytes(default_lbryum_dir)
+if 'darwin' in sys.platform:
+    platform = DARWIN
+    dirs = _get_old_directories(DARWIN)
+elif 'win' in sys.platform:
+    platform = WINDOWS
+    if os.path.isdir(user_data_dir('lbryum', roaming=True)) and \
+       os.path.isdir(user_data_dir('lbrynet', roaming=True)):
+        dirs = _get_old_directories(WINDOWS)
```

Comment:
So instead of 
`dirs = _get_old_directories(WINDOWS)`
You want 
```
dirs['data'] = _get_old_directories(WINDOWS)['data']
dirs['lbryum'] = _get_old_directories(WINDOWS)['lbryum']
dirs['download'] = _get_old_directories(WINDOWS)['download']
```
Or am I misunderstanding?